### DISCUSSION TITLE
Use :e command

### DISCUSSION THREAD

**Discussion by ForgetAll** [JetBrains/ideavim#464] [Ideas]
When I use vim to develop, after I know project well, I prefer use :e command to open other file in the project. Even sometimes use this command to open some file in other dir. I need hint after enter :e and enter tab, can plugin show file list about what I type? 

![image](https://user-images.githubusercontent.com/15844656/154431850-7e056f92-99c3-41df-8739-2be6bc562f8c.png)
Just like the image.

**Comment by AlexPl292** [comment]
Hi, please track this issue https://youtrack.jetbrains.com/issue/VIM-268
